config.def.h

```diff
@@ -54,45 +61,75 @@ static const Layout layouts[] = {
 #define SHCMD(cmd) { .v = (const char*[]){ "/bin/sh", "-c", cmd, NULL } }
 
 /* commands */
-static char dmenumon[2] = "0"; /* component of dmenucmd, manipulated in spawn() */
-static const char *dmenucmd[] = { "dmenu_run", "-m", dmenumon, "-fn", dmenufont, "-nb", col_gray1, "-nf", col_gray3, "-sb", col_cyan, "-sf", col_gray4, NULL };
+static const char *dmenucmd[] = { "dmenu_run",  NULL };
 static const char *termcmd[]  = { "st", NULL };
+static const char *sLock[]  = { "slock", NULL };
+static const char *backlitInc[]  = { "sudo", "brightnessctl", "set", "+2", NULL };
+static const char *backlitDec[]  = { "sudo", "brightnessctl", "set", "2-", NULL };
+static const char *volToggle[]  = { "pactl", "set-sink-mute", "0", "toggle", NULL };
+static const char *volDown[]  = { "pactl", "set-sink-volume", "0", "-2%", NULL };
+static const char *volUp[]  = { "pactl", "set-sink-volume", "0", "+1%", NULL };
```

Comment:
Additional dependancies are required. What if the user does not have PulseAudio?